### PR TITLE
Supertest

### PR DESCRIPTION
**Pull request by donia98**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,10 @@
     "test": "test"
   },
   "scripts": {
-    "dbtest": "NODE_ENV=test node test/test_db.js | tap-spec",
-    "start": "nodemon src/index.js"
+    "dbtest": "NODE_ENV=test tape test/test.js | tap-spec",
```
Comment (FarahZaqout) [inline package.json]: this script will fail. The file is ```test_db```, not ```test```.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (donia98) [reply package.json]: I solved it. could you approve the pull request or marge it?

File: src/controllers/sign_in.js [hunk 2]
```diff
@@ -48,7 +47,7 @@ exports.post = (req, res, next) => {
           })
           .catch(err => res.send({ err: `Error in password hash and ${err}` }));
       } else {
-        res.send({ err: 'invalid username or password' });
```
Comment (FarahZaqout) [inline src/controllers/sign_in.js:51]: The removed message was the proper one. Length is < 0 will only confuse the user.

File: src/middlewares/checkauth.js [hunk 3]
```diff
@@ -2,13 +2,15 @@ const { verify } = require('jsonwebtoken');
...
 exports.isSignIn = (req, res, next) => {
   req.isUser = false;
-  const { jwt } = req.cookies;
-  if (jwt) {
-    verify(jwt, process.env.SECRET, (err, data) => {
-      if (err) return;
-      req.isUser = true;
-      req.data = data;
-    });
+  if (req.cookies) {
```
Comment (FarahZaqout) [inline src/middlewares/checkauth.js:5]: req.cookies will always be true. This line is unnecessary.

**Review by FarahZaqout — APPROVED** [review]

**Pull request merged**